.github/workflows/openapi-diff.yml

```diff
@@ -0,0 +1,49 @@
+name: acceptance
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'api-docs/**'
+  pull_request:
+    paths:
+      - 'api-docs/**'
+
+jobs:
+    diff:
+      if: ${{ github.ref != 'refs/heads/master' }}
+      runs-on: ubuntu-latest
```

Comment:
Can we use `ubuntu-16.04` instead of latest?

Reply:
@billyvg I don't mind, but is there a reason? 16.04 stops getting updates April 2021, so wouldn't it be a future security risk?

Reply:
Prefer explicit versions than having to guess what `latest` means

Reply:
converted to `ubuntu-18.04`